Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py

```diff
@@ -4,151 +4,314 @@
+                    indicator_obj['type'] = eachtype
+                    break
+
+            data = self.get_recursively(eachres, 'value')
+
+            ind_val = {}
+            for eachindicator in data:
+                typeval = auto_detect_indicator_type(eachindicator)
+                if typeval:
+                    ind_val[typeval] = eachindicator
+
+            if len(data) == 1:
+                indicator_obj['value'] = str(data[0])
+            elif indicator_obj['type'] in list(ind_val.keys()):
+                indicator_obj['value'] = str(ind_val[indicator_obj['type']])
+            elif len(ind_val) != 0:
+                indicator_obj['type'] = list(ind_val.keys())[0]
+                indicator_obj['value'] = ind_val[list(ind_val.keys())[0]]
+
+            if eachres.get('indicators'):
+                for eachindicator in eachres.get('indicators'):
+                    indicator_obj['title'] = eachindicator.get('title')
+                    indicator_obj['time'] = eachindicator.get('timestamp')
+            else:
+                for eachindicator in eachres.get('ttps').get('ttps'):
+                    indicator_obj['title'] = eachindicator.get('title')
+                    indicator_obj['time'] = eachindicator.get('timestamp')
+
+            indicator_obj['rawJSON'] = eachres
+            indicators.append(indicator_obj)
+
+        return indicators
+
+    def parse_to_json(self, content):
+        """
+        Parse the feed response to JSON
+        :param content: xml data to parse
+        """
+        if content:
+            stix_dict = STIXPackage.from_xml(etree.XML(content)).to_dict()  # parse to dictionary
+            return stix_dict
+        else:
+            return {}
+
+
+    def get_taxii(self, args):
         """
         Fetch Taxii events for the given parameters
-        :param method: Requests method to be used
-        :param taxiiurl: API URL Suffix to be used
-        :param params:
+        :param args: arguments which would be used to fetch feed
         :return:
         """
+        taxii_data = []
+        save_fetch_time = None
+        count = 0
+        try:
+            for data in self.__fetch(args.get('begin'), args.get('end'), args.get('collection')):
+                response = self.parse_to_json(data)
+
+                if response.get('indicators') or False:
+                    content = response.get('indicators')
+                elif response.get('ttps') or False:
+                    content = response.get('ttps').get('ttps')
+                else:
+                    raise ValueError(f"Last fetch time retrieval failed.")
+
+                for eachone in content:
+                    save_fetch_time = parser.parse(eachone['timestamp']).replace(tzinfo=pytz.UTC).strftime(
+                        DATETIME_FORMAT)
+
+                taxii_data.append(response)
+
+                count += 1
+                if count == arg_to_number(args.get('limit', 0)):
```

Comment:
When a limit is not provided, the condition will not be met.
```suggestion
                if count == arg_to_number(args.get('limit', 1)):
```